src/panels/events.js

```diff
@@ -1,14 +1,13 @@
 var React = require('react');
 var $ = require('jquery');
-var ICAL = require('ical.js');
 var moment = require('moment');
+var momentTimezone = require('moment-timezone');
```

Comment:
The `momentTimezone` variable is never used, but you call `tz` on the moment objects below, which if I'm not mistaken only works because of a side effect of requiring moment-timezone. Since moment-timezone is a superset of moment, it would be clearer to remove the `require('moment')` and just write `var moment = require('moment-timezone')`.